src/utils/task-run-filter-transforms.ts

```diff
@@ -36,7 +36,11 @@ export const convertFilterToKubearchiveSelectors = (
   filterBy: TaskRunSelector,
 ): Pick<WatchK8sResource, 'fieldSelector' | 'selector'> => {
   const fieldSelectors: Record<string, string> = {};
-  if (filterBy.filterByName) fieldSelectors['metadata.name'] = filterBy.filterByName;
+  if (filterBy.filterByName) {
+    // e.g. name=*e2e*
+    // Matches resources whose names contain "e2e" anywhere
+    fieldSelectors.name = `*${filterBy.filterByName}*`;
+  }
```

Comment:
both task-run-filter-transformers.ts and pipeline-run-filter-transforms.ts has same utils, can we remove the redundant and keep only one.

Reply:
done, I extracted the logic to a new file called `kubearchive-filter-transforms` and started using it in the codebase :)